R/z_utils_render_footnotes.R

```diff
@@ -409,7 +413,12 @@ resolve_footnotes_styles <- function(data, tbl_type) {
         tbl$fs_id <- process_footnote_marks(tbl$fs_id, marks = footnote_marks)
       }
 
-      tbl <- vctrs::vec_rbind(tbl_no_loc, tbl)
+      if(footnote_order == "marks_first"){
+        tbl <- vctrs::vec_rbind(tbl, tbl_no_loc)
+      }else if(footnote_order == "marks_last"){
```

Comment:
@thebioengineer I don't know if to just do this as else() rather than else if as marks_last is the default and the function is written so if anything other than marks_first and marks_last is input it will fail before it gets to this point.